public/docs/dart/latest/guide/lifecycle-hooks.jade

```diff
@@ -3,9 +3,16 @@ extends ../../../ts/_cache/guide/lifecycle-hooks.jade
 block includes
   include ../_util-fns
 
-block optional-interfaces
-  //- n/a for Dart
+block other-angular-subsystems
+  :marked
+    The router, for instance, also has its own [router lifecycle
+    hooks](router.html#router-lifecycle-hooks) that allow us to tap into
+    specific moments in route navigation.
+    A parallel can be drawn between `ngOnInit` and `routerOnActivate`.  Both are
+    prefixed so as to avoid collision, and both run right when a component is
+    'booting' up.
```

Comment:
Agreed. Fixed.

(For the record, I just copied this from the pre-RC5 prose. :)